tools/fonts/fontgen_smufl.js

```diff
@@ -45,11 +45,11 @@ function toVFPath(glyph) {
   const pathStr = ops.join(' ');
 
   return {
-    x_min: bb.x1,
-    x_max: bb.x2,
-    y_min: bb.y1,
-    y_max: bb.y2,
-    ha: bb.y2 - bb.y1,
+    x_min: Math.round(bb.x1),
```

Comment:
We round these values to nearest whole integer? Is the output any different if these are not rounded? Or rounded to two or three decimal points?

Reply:
There is no difference, that I can see. All the other fonts are rounded and I wanted to keep it consistent.